src/main.rs

```diff
@@ -36,6 +38,13 @@ fn main() {
                     .help("Just print the hex values, not colour previews")
```

Comment:
This comment will need updating (more generally we should probably search for "hex" and replace any mention of it).

This includes putting an example in the README.